src/Microsoft.Azure.Functions.ExtensionBundle/extensions.json

```diff
@@ -66,6 +66,30 @@
       "orchestrationclient"
     ]
   },
+  {
+    "id": "Microsoft.DurableTask.SqlServer.AzureFunctions",
+    "majorVersion": "0",
+    "name": "SqlDurabilityProviderStartup",
+    "bindings": [
+      "activitytrigger",
+      "orchestrationtrigger",
+      "entitytrigger",
+      "durableclient",
+      "orchestrationclient"
+    ]
+  },
+  {
+    "id": "Microsoft.Azure.DurableTask.Netherite.AzureFunctions",

```

Comment:
In Connor's PR, this `id` didn't have the `.AzureFunctions` suffix, but I think that was an omission on his part